docs/src/components/Github/index.tsx

```diff
@@ -1,47 +1,68 @@
 import React from "react";
 import { useStaticQuery, graphql } from "gatsby";
+import styled from "styled-components";
+import { Alert, Text } from "@kiwicom/orbit-components";
 
 import Contributor from "./Contributor";
 
+const Grid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(60px, 1fr));
+  grid-gap: ${({ theme }) => theme.orbit.spaceXSmall};
+`;
+
 export interface IContributor {
   name?: string;
   id?: string;
-  avatarUrl?: string;
+  avatar_url?: string;
   bio?: string;
-  websiteUrl?: string;
-  twitterUsername?: string;
+  blog?: string;
+  twitter_username?: string;
   url?: string;
 }
 
 const ContributorsComponent = () => {
-  const { github } = useStaticQuery(
+  const { allContributor } = useStaticQuery(
     graphql`
       query CollaboratorsQuery {
-        github {
-          repository(name: "orbit", owner: "kiwicom") {
-            collaborators {
-              nodes {
-                bio
-                avatarUrl
-                id
-                name
-                url
-                websiteUrl
-                twitterUsername
-              }
-            }
+        allContributor {
+          nodes {
+            id
+            avatar_url
+            bio
+            blog
+            twitter_username
+            name
+            url
           }
         }
       }
     `,
   );
 
-  return (
-    <>
-      {github.repository.collaborators.nodes.map(({ id, ...info }) => (
+  const githubDocs = (
+    <a
+      href="https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token"
+      aria-label="creating-a-personal-access-token"
+    >
+      Please create one
+    </a>
+  );
+
+  return allContributor.nodes && allContributor.nodes.length > 0 ? (
+    <Grid>
+      {allContributor.nodes.map(({ id, ...info }) => (
         <Contributor key={id} id={id} {...info} />
       ))}
-    </>
+    </Grid>
+  ) : (
+    <Alert type="warning">
+      <Text>
+        Contributors component is not shown, because you are missing github personal access token
+        {githubDocs}
+        Check the terminal for more details
```

Comment:
```suggestion
        Contributors component is not shown, because you are missing github personal access token.{" "}
        {githubDocs}{" "}
        Check the terminal where you started the site for more details.
```